src/mkdocstrings/plugin.py

```diff
@@ -103,7 +107,8 @@ class MkdocstringsPlugin(BasePlugin):
 
     def __init__(self) -> None:
         super(MkdocstringsPlugin, self).__init__()
-        self.mkdocstrings_extension = None
+        # stop mypy from complaining about NoneType mismatches
+        self.mkdocstrings_extension = MkdocstringsExtension(config={})
```

Comment:
I'd prefer a `# type: ignore` comment here as well, and reverting the change in the class init method. Having an uninitialized extension seems fragile.